lms/djangoapps/certificates/views.py

```diff
@@ -245,6 +245,16 @@ def render_html_view(request):
     context = {}
     configuration = CertificateHtmlViewConfiguration.get_config()
     context = configuration.get('default', {})
+    active_certificate = None
+
+    def _get_certificate_signatories(active_cert):
```

Comment:
Let's change the name of the 'active_cert' argument to 'certificate' for this method -- it does not care whether a certificate is active or not, it just gets the signatories.